past_meetings.html

```diff
@@ -19,8 +19,8 @@ <h2><a href="{% url 'meeting' meeting.id %}">{{ meeting }}</a></h2>
       {% if topic.length %}
         ({{ topic.length }} Minutes)<br />
       {% endif %}
-      By: {% for presentor in topic.presentors.all %}
-            {{presentor.name}}{% if not forloop.last %}, {% endif %}
+      By: {% for presenter in topic.presenters.all %}
```

Comment:
```suggestion
      By: {% for presentor in topic.presentors.all %}
```
Rolling back so it doesn't conflict with our deleting that file